### DISCUSSION TITLE
Can't display video in the table

### DISCUSSION THREAD

**Discussion by MohamedAshour119** [filamentphp/filament#13053] [Help]
### Package

Table builder

### Package Version

v3.0.0

### How can we help you?

I can't display video in the table, what is the problem?
![Screenshot_1](https://github.com/filamentphp/filament/assets/131293252/ab5fbe50-d6e1-4f6b-a06c-ddad94b1a1c5)

**Comment by leandrocfe** [comment]
Wouldn't make more sense show a thumb instead of render the video?
You could save an image as thumb and show it in the table 
https://filamentphp.com/docs/3.x/tables/columns/image
